nipype/utils/filemanip.py

```diff
@@ -295,8 +269,23 @@ def _generate_cifs_table():
         (line.split()[2:5:2] for line in output.splitlines()),
         key=lambda x: len(x[0]),
         reverse=True)
-    cifs_paths = [path for path, fstype in mount_info if fstype == 'cifs']
 
+    # find which mount points are CIFS
+    # init to empty list
+    cifs_paths = []
+        
+    try:    
+	    for  path_and_fstype in mount_info:
+			# need to check for tables that have only path and no fstype
+			if len(path_and_fstype) == 2:
+				# if this entry is cifs, add it to list
+				if path_and_fstype[1] == 'cifs':
+					cifs_paths.append(path_and_fstype[0])			
+			else:
+				fmlogger.debug('mount file system types not described by fstype')
+    except:
+		fmlogger.debug('mount file system type check for CIFS error')
+		return []
```

Comment:
What about just:

```Python
cifs_paths = [entry[0] for entry in mount_info if len(entry) > 1 and entry[1] == 'cifs']
```

Unless there's a case where a CIFS mount would fall into the case where `len(entry) == 1`, the warning seems like it'll be noise in the log.

Under what cases were you hitting this error, by the way?

Reply:
Most of the removed lines was just a mistake on my part because I had moved the file from an older version of the nipype source. I think those lines should now be back in. Really the only part I introduced was the cifs table checking. It seems to work with my own mount table which now has about a dozen entries. This seems to be run only on the initial import, so the debug lines shouldn't be too great and should only be for those mount entries that only have a single item length.

Reply:
I mean what does your `mount` output look like that doesn't have 5 entries in a given row?

Reply:
the "mount" output and the mount_info in python looks like this:

% mount
/dev/disk2 on / (hfs, local, journaled)
devfs on /dev (devfs, local, nobrowse)
map -hosts on /net (autofs, nosuid, automounted, nobrowse)
map auto_home on /home (autofs, automounted, nobrowse)
map -fstab on /Network/Servers (autofs, automounted, nobrowse)
/dev/disk4s2 on /Volumes/DG TimeMachine  (hfs, local, nodev, nosuid, journaled)
/dev/disk3s2 on /Volumes/MyBookData (hfs, local, nodev, nosuid, journaled)
afni:/elrond0 on /Volumes/afni (nfs)
afni:/var/www/INCOMING on /Volumes/INCOMING (nfs)
afni:/fraid on /Volumes/afni (nfs, asynchronous)
boromir:/raid.bot on /Volumes/raid.bot (nfs)
elros:/volume2/AFNI_SHARE on /Volumes/AFNI_SHARE (nfs)
map -static on /Volumes/safni (autofs, automounted, nobrowse)
map -static on /Volumes/raid.top (autofs, automounted, nobrowse)
/dev/disk1s3 on /Volumes/Boot OS X (hfs, local, journaled, nobrowse)

ipdb> mount_info
[['/Volumes/MyBookData', 'local,'], ['/Volumes/AFNI_SHARE'], ['/Volumes/INCOMING'], ['/Volumes/raid.bot'], ['/Volumes/afni'], ['/Volumes/afni', 'asynchronous)'], ['/Volumes/Boot', 'X'], ['/Volumes/DG', '(hfs,'], ['/dev', 'local,'], ['on', '(autofs,'], ['on', '(autofs,'], ['on', '(autofs,'], ['on', '(autofs,'], ['on', '(autofs,'], ['/', 'local,']]